### ISSUE TITLE
Subtract relations from Wikipedia

### ISSUE THREAD

**Issue by Sandr0x00** [MusicConnectionMachine/api#35]
Continuing this discussion (https://github.com/MusicConnectionMachine/RelationshipsG3/issues/22) where it belongs: here :)
@Henni @martomi

**Comment by RBirkeland** [comment]
Some points to have in mind:

1. Because we use the certainty and occurrence to filter out non trustworthy information or irrelevant information, how can we distinguish between these "fun facts"/non-Wikipedia relations from completely irrelevant information?

2. Also how does @martomi @vviro @gyachdav @kordianbruck want use this information?

**Comment by gyachdav** [comment]
The idea here is to filter out those named entities that are found in our relationships database and can also be found on the Wikipedia info box. E.G. most likely we would find the relationship Beethoven born in Bonn, Germany. Wikipedia already has that relationship in the info box section. So the outcome of this issue will be to filter out this tired tidbit from the knowledge base we are creating.

**Comment by RBirkeland** [comment]
Right, so you want to distinguish between "common knowledge", something that everyone knows, and less known facts?

I think this can simply be implemented by query results that have occurrence lower than a certain threshold. The obvious drawback here is I stated before, that low occurrence may also mean low trustworthiness.
